.npmignore

```diff
@@ -10,3 +10,5 @@ webpack.config.js
 build-utils/
 coverage/
 test/
+docs/
```

Comment:
also need to ignore/exclude `site/` (?)

The directory `site/` is generated by `npm run doc`